README.md

```diff
@@ -1,2 +1,27 @@
-# bpmn-visualization-R
+<h1 align="center">BPMN Visualization - R Package</h1>
+<div align="center">
+    <p align="center">
+        <a href="https://github.com/process-analytics/bpmn-visualization-r/releases">
+          <img alt="GitHub release (latest by date including pre-releases)" src="https://img.shields.io/github/v/release/process-analytics/bpmn-visualization-r?label=changelog&include_prereleases"> 
+        </a>
+        <a href="https://github.com/process-analytics/bpmn-visualization-r/actions">
+          <img alt="Build" src="https://github.com/process-analytics/bpmn-visualization-r/workflows/Build/badge.svg"> 
+        </a> 
```

Comment:
⚠️ this icon will be broken until we have a GH WF configured in this repo

Reply:
Yes, I though to configure the worklow later ^^